packages/loader/container-loader/src/container.ts

```diff
@@ -528,6 +531,16 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
         this.removeAllListeners();
     }
 
+    public getPendingLocalState(): string {
+        assert(this.resolvedUrl !== undefined && this.resolvedUrl.type === "fluid");
```

Comment:
Could you please add assert (above it) that container is attached?
I believe it's same thing as what you assert, but more on point.